servicemesh-release-notes/topics/rn-new-features.adoc

```diff
@@ -5,6 +5,10 @@ Feature – Describe the new functionality available to the customer.  For enhan
 Reason – If known, include why has the enhancement been implemented (use case, performance, technology, etc.).   For example, showcases integration of X with Y, demonstrates Z API feature, includes latest framework bug fixes.  There may not have been a 'problem' previously, but system behaviour may have changed.
 Result – If changed, describe the current user experience
 ////
+= New features Technology Preview 4
+
+The release of {ProductName} adds support for Kiali 0.9.x.
```

Comment:
Might make sense here to mention that tp4 is based on Istio 1.0.3

Reply:
@brian-avery I've made this change.